views/address.tmpl

```diff
@@ -26,10 +26,10 @@
                         {{range .Transactions}}
                         <tr>
                             <td><a href="../tx/{{.TxID}}" class="hash collapse">{{.TxID}}</a></td>
-                            <td>{{.Value | getAmount}}</td>
+                            <td>{{.Value | float64Commaf}}</td>
```

Comment:
or you can put "DCR" at the end of this <td> instead of in the column header, but I like not having it on every single amount, as you made it.